# Product type
string type

# Whether or not the product is faulty
bool is_faulty

